medusa/notifiers/emailnotify.py

```diff
@@ -20,6 +20,21 @@
 
 
 class Notifier(object):
+
+    # Possible patterns:
```

Comment:
Use docstring for this?

Reply:
But it doesn't describe the notifier class, just the possible patterns for the regex pattern below.

Reply:
Then we make a short class docstring?